src/installerOmod.cpp

```diff
@@ -46,7 +46,7 @@ QString InstallerOMOD::description() const
 
 MOBase::VersionInfo InstallerOMOD::version() const
 {
-  return MOBase::VersionInfo(1, 0, 0, MOBase::VersionInfo::RELEASE_PREALPHA);
+  return MOBase::VersionInfo(1, 1, 0, 0);
```

Comment:
I'm pretty sure the last field here should be one of the `MOBase::VersionInfo::RELEASE_XXXXX` enum values.

Reply:
VersionInfo has several variants.  This is switching from
`VersionInfo(int major, int minor, int subminor, ReleaseType releaseType = RELEASE_FINAL);`
to
`VersionInfo(int major, int minor, int subminor, int subsubminor, ReleaseType releaseType = RELEASE_FINAL);`.  Doesn't really matter.